resources/add_language.py

```diff
@@ -0,0 +1,190 @@
+"""
+A script to add a new language to the gritql metavariable-grammars.
+"""
+
+# Future imports (must occur at the beginning of the file):
+from __future__ import annotations  # https://www.python.org/dev/peps/pep-0585/
+
+# Standard library imports:
+import os
+import re
+import json
+from argparse import Namespace, ArgumentParser
+from subprocess import run, check_output, CalledProcessError
+
+resources_path = os.path.dirname(os.path.realpath(__file__))
+repo_path = os.path.abspath(os.path.join(resources_path, os.pardir))
+
+# https://regex101.com/r/nRTpjv/1
+ALL_LANGUAGES = re.compile(
+    r"""\/{51}
+\/\/ Constants
+\/{51}
+const allLanguages = (?P<all_languages>\[
+  (\s*"[\w\-]+",\n?)+
+  (?P<last_language>\s*"[\w\-]+",)
+\])""",
+)
```

Comment:
We should use GritQL for this.

Reply:
Good point. I don't really know GritQL well enough, yet. Whereas regex, I know all too well 🤣

Reply:
Thankfully this one is very simple:
```
`const allLanguages = [$langs]` where { $langs += `"foo"`}
```

Reply:
Very nice. Thanks. Might be worthwhile for me to spend a bit of time learning GritQL after this PR then!

Reply:
I'm just going to add "At least 10 test cases, including rewrites and metavariables." as you mentioned on the issue description. From your perspective, once I do that (assuming they all pass or are fixed as needed) would this PR be ready to merge? Or have I missed something else?

Reply:
I think I'd opt to remove this file, since it will potentially require maintenance and doesn't really make things much faster/easier as-is.